src/worker.js

```diff
@@ -134,17 +131,30 @@ const transcribe = async (
         last.tokens = [...item[0].output_token_ids];
 
         // Merge text chunks
-        // TODO optimise so we don't have to decode all chunks every time
-        let data = transcriber.tokenizer._decode_asr(chunks_to_process, {
+        let data = transcriber.tokenizer._decode_asr([last], {
             time_precision: time_precision,
             return_timestamps: true,
             force_full_sequences: false,
         });
+        // chunk_length_s should be isDistilWhisper ? 20 : 30, but timestamps are off with that logic
```

Comment:
I am getting chunks of 20s even using the base model - so I hardcode this to prevent timestamps being off